features/step_definitions/dishes_steps.rb

```diff
@@ -1,6 +1,12 @@
 Given(/^the following dishes exist:$/) do |table|
   table.hashes.each do |dish|
-    FactoryGirl.create(:dish, dish)
+    if dish[:category]
+      category = Category.find_by(name: dish[:category])
+      hash = dish.except!(dish[:category]).merge(category: category)
+      FactoryGirl.create(:dish, hash)
+    else
+      FactoryGirl.create(:dish, dish)
+    end
```

Comment:
I'm honored @j0sv  😀

Reply:
@tochman It was a nice piece of code so why not use it. 🥇